docs/docs/Get-Started/get-started-installation.md

```diff
@@ -79,6 +79,25 @@ To reinstall Langflow and all of its dependencies, add the `--force-reinstall` f
 python -m pip install langflow --force-reinstall
 ```
 
+## Install Langflow Desktop
+
+:::important
+Langflow Desktop is available only for macOS.
+:::
+
+**Langflow Desktop** is a desktop version of Langflow.
```

Comment:
Does Langflow Desktop have all of the same functionality as OSS LF? If not, I suggest adding a topic that explains the differences.

Reply:
Good question, asking

Reply:
Added section on version management with the desktop application